internal/db/schema/migrations/postgres/2/01_job.up.sql

```diff
@@ -0,0 +1,87 @@
+begin;
+
+create table job (
+     private_id wt_private_id primary key,
+     name wt_name not null,
+     description wt_description not null,
+     code text not null
+         constraint code_too_short
+             check (length(trim(value)) > 0)
+         constraint code_too_long
+             check (length(trim(value)) < 128),
+     next_scheduled_run timestamp not null,
+
+     constraint job_name_code_uq
+         unique(name, code)
+);
+
+comment on table job is
+    'job is a table where each row represents a unique job that can only have one running instance at any specific time.';
+
+create trigger immutable_columns before update on job
+    for each row execute procedure immutable_columns('private_id', 'name', 'code');
+
+create table job_run_status_enm (
+    name text not null primary key
+        constraint only_predefined_job_status_allowed
+            check(name in ('running', 'completed', 'failed', 'interrupted'))
+);
+
+comment on table job_run_status_enm is
+    'job_run_status_enm is an enumeration table where each row contains a valid job run state.';
+
+insert into job_run_status_enm (name)
+    values
+    ('running'),
+    ('completed'),
+    ('failed'),
+    ('interrupted');
+
+create table job_run (
+     id serial primary key,
+     job_id text not null
```

Comment:
Why is the column type (`text`) different from the column type in the referenced table (`wt_private_id`)?